lib/xandra/protocol.ex

```diff
@@ -502,16 +515,17 @@ defmodule Xandra.Protocol do
     value
   end
 
-  def decode_response(%Frame{kind: :event, body: body}, nil, _options) do
+  def decode_response(%Frame{kind: :event, body: body, atom_keys?: atom_keys?}, nil, _options) do
     decode_string(event <- body)
     "STATUS_CHANGE" = event
-    decode_string(effect <- body)
+    decode_string_or_atom(effect <- body, atom_keys?)
     {address, port, <<>>} = decode_inet(body)
     %StatusChange{effect: effect, address: address, port: port}
   end
 
-  def decode_response(%Frame{kind: :result, body: body}, %kind{} = query, options)
+  def decode_response(%Frame{kind: :result, body: body, atom_keys?: atom_keys?}, %kind{} = query, options)
       when kind in [Simple, Prepared, Batch] do
+    options = Keyword.merge([atom_keys?: atom_keys?], options)
```

Comment:
```elixir
options = Keyword.put(options, :atom_keys, atom_keys?)
```

Reply:
Hmm, my thinking here was to allow people to override it via options if they wanted to. Perhaps I should use `put_new` instead?